src/main/python/ayab/plugins/ayab_plugin/ayab_image.py

```diff
@@ -19,17 +19,19 @@
 #    https://github.com/AllYarnsAreBeautiful/ayab-desktop
 
 from PIL import Image
+from bitarray import bitarray
 import numpy as np
 
+MACHINE_WIDTH = 200
```

Comment:
This is declared in ayab.py, ayab_control.py and test_ayab_control.py already. Perhaps we could put this into a dedicated .py and include it.

Reply:
Yes, there are a couple of other global constants I would like to introduce as well. I will do that in a separate commit.